Applications/Games/Space Engineers/Steam/script.js

```diff
@@ -0,0 +1,38 @@
+include("engines.wine.quick_script.steam_script");
+include("engines.wine.verbs.dotnet472");
+include("engines.wine.verbs.vcrun2017");
+include("engines.wine.verbs.dxvk");
+include("engines.wine.verbs.faudio");
+include("engines.wine.plugins.override_dll");
+include("utils.functions.filesystem.files");
+
+var installerImplementation = {
+    run: function () {
+        new SteamScript()
+            .name("Space Engineers")
+            .editor("Keen Software House")
+            .author("Zemogiter")
+            .appId(244850)
+            .wineVersion("4.9")
+            .wineDistribution("upstream")
+            .wineArchitecture("amd64")
+            .preInstall(function (wine, wizard) {
+                wine.dotnet472();
+                wine.vcrun2017();
+                wine.DXVK();
+                wine.faudio();
```

Comment:
This should not be necessary with wine 4.9 since it uses faudio by default

Reply:
I've tried the same but without faudio verb and it crashed.

Reply:
Ok weird.